Assets/Source/Components/UI/HealthBarComponent.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Assets.Source.Components.Base;
+using UnityEngine;
+
+namespace Assets.Source.Components.UI
+{
+    public class HealthBarComponent : ComponentBase
+    {
+        public Animator Animator { get; private set; }
+
+        public override void ComponentAwake()
+        {
+            Animator = GetRequiredComponent<Animator>();
+
+            base.ComponentAwake();
+        }
+
+        public void StopHighlight()
+        {
+            Animator.SetBool("Highlight", false);
```

Comment:
super minor and dumb, but for animations i've always used the all lowercase / underscore naming convention for animation triggers. 

- [ ] Rename to "highlight" or whatever it needs to be renamed to after the refactor

Reply:
okay no worries i made it lowercase